apps/web/pages/api/auth/signup.ts

```diff
@@ -71,6 +71,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   });
 
   // If user has been invitedTo a team, we accept the membership
+  // TODO check memberships instead of single invitedTo
```

Comment:
Is this a todo we have to fix before merge?

Reply:
Not needed anymore

Reply:
Should  we remove it then?
```suggestion

```